src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java

```diff
@@ -71,15 +79,22 @@ public void visitToken(DetailAST ast) {
         final DetailAST defaultGroupAST = ast.getParent();
         //default keywords used in annotations too - not what we're
         //interested in
+        if (ast.getPreviousSibling() != null) {
+            TOKEN_TYPE.add(ast.getPreviousSibling().getType());
```

Comment:
We do alot of adding but no removes or clears. If we run this on multiple files, this list will become bigger and bigger with no end in sight and probably lead to false handling.
It is also not clear why we need this field and just get tokens down below on line 94.